### ISSUE TITLE
TPS is not grouping by second but rather milisecond

### ISSUE THREAD

**Issue by damies13** [damies13/rfswarm#155]
**Describe the bug**
Excerpt from discord chat when bug was discovered by @Anker101

---
Anker01 — 17/06/2023 00:53

---
i also have another question  about report 
I run a test with 25 robots per scenario for 2 minutes , one scenario is doing get calls and the other one , post calls
And the report when i read the RPS graph it mostly always a 1 rps 
But in count data table i have much more requests executed 
See screen (edited)
![image](https://github.com/damies13/rfswarm/assets/15232871/cf889b09-fbe9-43cb-8e11-3f4c74b46e7e)

---
damies13 — 17/06/2023 09:46

---
Regarding the RPS (TPS), good catch there's a bug that would have taken me a lot longer to find. can you log a bug report for it? At the moment the generated SQL selects end_time as 'Time' ,  it should select floor(end_time) as 'Time' ,  as the end_time column is a float so stores fractions of a second, using floor will group all the transactions from the same second together. The same bug applies to Total TPS

**Comment by damies13** [comment]
TPS SQL is:
```
SELECT 
 	end_time as 'Time' , 
	count(result) as 'Value' , 
	result_name as [Name] 
FROM Results 
WHERE 
	[Name] NOT LIKE 'Exception in thread%' 
GROUP by 
 	end_time , 
	result_name , 
	result 
ORDER by 
 	end_time, 
	result DESC, 
	count(result) DESC

```
Should be:
```
SELECT 
	floor(end_time) as 'Time' , 
	count(result) as 'Value' , 
	result_name as [Name] 
FROM Results 
WHERE 
	[Name] NOT LIKE 'Exception in thread%' 
GROUP by 
	floor(end_time) ,
	result_name , 
	result 
ORDER by 
	floor(end_time) ,
	result DESC, 
	count(result) DESC

```

Likewise Total TPS SQL is generated as:
```
SELECT 
 	end_time as 'Time', 
	count(result) as 'Value' , 
	result as [Name] 
FROM Results 
GROUP by 
 	end_time , 
	result 
ORDER by 
 	end_time, 
	count(result) DESC
	
	
-- 1679302277.668
-- 1679302280.458
```
should be:
```
SELECT 
	floor(end_time) as 'Time' , 
	count(result) as 'Value' , 
	result as [Name] 
FROM Results 
GROUP by 
	floor(end_time) ,
	result 
ORDER by 
	floor(end_time) ,
	count(result) DESC
	
	
-- 1679302277.668
-- 1679302280.458
```

- [ ] This may also impact the graphs in manager, needs to be checked
- [ ] This may also impact the data table in reporter, needs to be checked

**Comment by damies13** [comment]
Fix merged into Release v1.1.4 Branch
